### ISSUE TITLE
Unhandled rejection TypeError: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by ide** [tjunnone/npm-check-updates#113]
With version 2.0.1 I get this error:

```
$ ncu
Unhandled rejection TypeError: Cannot convert undefined or null to object
    at Function.keys (native)
    at Object.numParts (/Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/lib/version-util.js:24:49)
    at /Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/lib/versionmanager.js:78:32
    at /Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:7113:30
    at /Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2275:27
    at /Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3049:15
    at baseMap (/Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2274:7)
    at Function.sortBy (/Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:7112:20)
    at baseWrapperValue (/Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2768:30)
    at LodashWrapper.wrapperValue (/Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:6124:14)
    at upgradeDependencyDeclaration (/Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/lib/versionmanager.js:80:10)
    at wrapper (/Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3533:19)
    at /Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:8201:21
    at /Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3395:24
    at /Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:3073:15
    at baseForOwn (/Users/ide/.nvm/versions/io.js/v2.5.0/lib/node_modules/npm-check-updates/node_modules/lodash/index.js:2046:14)
```

The offending part of my package.json seems to be this:

```js
  "devDependencies": {
    "eslint": "^1.0.0-rc-2",
  }
```

One thing that may have changed is that eslint 1.0.0 (non-RC) was published just 3 hours ago.

**Comment by metaraine** [comment]
Thank you for reporting! I will investigate.

**Comment by metaraine** [comment]
Yes, this appears to be a bug in one of our dependencies, `semver-utils`. I will work on a patch.

**Comment by metaraine** [comment]
Fixed in v2.0.02
